package.json

```diff
@@ -31,7 +32,7 @@
     "url": "git+https://github.com/Esri/calcite-components.git"
   },
   "dependencies": {
-    "@a11y/focus-trap": "^1.0.2",
+    "@a11y/focus-trap": "git+https://github.com/patrickarlt/focus-trap.git#conditional-define-build",
```

Comment:
did you open a pr form this branch already?

Reply:
Yup. This particular branch just has the built files in it already vs the one I PR'ed https://github.com/andreasbm/focus-trap/pull/1.